scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
+    box-shadow: $crm-form-control-shadow;
+    border: 1px solid $crm-darker-grayblue;
+    line-height: 1em;
+    padding: 8px 15px;
+    margin-right: -5px;
+    min-width: 180px;
+
+
+    &.active {
+      background-color: $gray-lighter;
+    }
+
+    &:first-child {
+      border-radius: 2px 0 0 2px;
+    }
+
+    &:last-child {
+      border-radius: 0 2px 2px 0;
+    }
+
+    input {
+      margin-right: 10px;
```

Comment:
I updated this to be like 
```
    input[type="radio"] {
      margin-right: 10px;
    }
```

Radio inputs require some right margin 

![sdasd](https://cloud.githubusercontent.com/assets/2423218/24747002/3ada33b2-1abb-11e7-8bc3-e7888761eaf7.png)